airflow/www/templates/airflow/dag.html

```diff
@@ -139,6 +139,9 @@ <h4 class="pull-right" style="user-select: none;-moz-user-select: auto;">
           <li><a href="{{ url_for('Airflow.code', dag_id=dag.dag_id, root=root) }}">
             <span class="material-icons" aria-hidden="true">code</span>
             Code</a></li>
+          <li><a href="{{ url_for('Airflow.audit_log', dag_id=dag.dag_id) }}">
```

Comment:
```suggestion
          <li><a href="{{ url_for('Airflow.audit_log', dag_id=dag.dag_id, root=root) }}">
```

Do we need to pass `root` in here? I don't fully understand what its used for but I see other views which extend `airflow.html` pass it in.